blueprints/APIv3.tex

```diff
@@ -461,7 +478,79 @@ \subsection{Relationships as Objects}
 has Servers as a property. Thus assignments are manipulated by manipulating that property. So the only endpoints necessary for fully
 defining and dealing with such relationships are \code{/delivery\_services} and \code{/servers}.
 
-\section{Data Model}
+\section{Authentication and Authorization}
+Authentication with the Traffic Ops API is done via the \code{/login} endpoint (detailed in Section \ref{sec:endpoint:login}) to obtain
+credentials, and thereafter by presenting those credentials in requests made to the server. The default authorization scheme is using a
+cookie named "mojolicious" which contains the authenticated user's username using SCRYPT. For more information, consult the
+\href{https://godoc.org/golang.org/x/crypto/scrypt}{documentation for the \code{golang.org/x/crypto/scrypt} package used to implement it}.
+Traffic Ops API endpoints MUST not only recognize this method of authentication, but also MUST check it before any other authentication
+scheme.\\
+
+\subsection{JWT Authentication}
+Alternatively, authentication can be done using OAuth2 Bearer tokens in JSON Web-Token (JWT) format\footnote{The intention of this section
```

Comment:
Might be good to specify which RFC apply to these JWT.  There are several for separate purposes.

Reply:
I did specify a couple later on - was it not a complete list/do you think I should move them to earlier in the section?

Reply:
I'd say just move it earlier.

Reply:
It actually looks like I was missing number 7519, but that should be fixed now. Better?

Reply:
Yep